### PR TITLE
Made postfix name configurable for Persistent&Volatile Path of plugins

### LINKED ISSUE #154: Add option to configure plugin to use data/persistent/volatile paths of 'main' callsign
For example, WebKitBrowser plugin has multiple 'alias' callsigns that use Shared Data/Persistent/Volatile paths derived from their name.

The request is to have a way to define/configure what 'alias' callsigns should be using.

### PR DESCRIPTION
**Pull request by HaseenaSainul**

### CODE REVIEW

File: Source/plugins/Configuration.h [hunk 1]
```diff
@@ -102,10 +112,12 @@ namespace Plugin {
             return (basePath + ClassName.Value() + '/');
         }
         string PersistentPath(const string& basePath) const {
-            return (basePath + Callsign.Value() + '/');
+            string postfixPath = (PersistentPathPostfix.IsSet() == true) ? PersistentPathPostfix.Value(): Callsign.Value();
```
Comment (pwielders) [inline Source/plugins/Configuration.h]: Can we also check for not empty.

File: Source/plugins/Configuration.h [hunk 2]
```diff
@@ -102,10 +112,12 @@ namespace Plugin {
...
+            return (basePath + postfixPath + '/');
         }
         string VolatilePath(const string& basePath) const {
-            return (basePath + Callsign.Value() + '/');
+            string postfixPath = (VolatilePathPostfix.IsSet() == true) ? VolatilePathPostfix.Value(): Callsign.Value();
```
Comment (pwielders) [inline Source/plugins/Configuration.h]: Can we also check for not empty.

**Review by pwielders — CHANGES_REQUESTED** [review]

File: Source/plugins/Configuration.h (hunk 1 above)
Comment (HaseenaSainul) [reply Source/plugins/Configuration.h]: Changes added

File: Source/plugins/Configuration.h (hunk 2 above)
Comment (HaseenaSainul) [reply Source/plugins/Configuration.h]: Changes added

**Review by pwielders — APPROVED** [review]

**Pull request merged**